cfme/services/myservice.py

```diff
@@ -166,117 +247,120 @@ def exists(self):
             return False
 
     def delete(self):
-        navigate_to(self, 'Details')
-        cfg_btn(
+        view = navigate_to(self, 'Details')
+        view.configuration.item_select(
             version.pick({
                 version.LOWEST: 'Remove Service from the VMDB',
                 '5.7': 'Remove Service'}),
-            invokes_alert=True)
-        sel.handle_alert()
-        flash.assert_success_message('Service "{}": Delete successful'.format(self.service_name))
+            handle_alert=True)
+        view = self.create_view(MyServicesView)
+        assert view.is_displayed
+        view.flash.assert_success_message(
+            'Service "{}": Delete successful'.format(self.name))
 
     def set_ownership(self, owner, group):
-        navigate_to(self, 'SetOwnership')
-        fill(set_ownership_form, {'select_owner': owner,
-                                  'select_group': group},
-             action=form_buttons.save)
-        flash.assert_success_message('Ownership saved for selected Service')
+        view = navigate_to(self, 'SetOwnership')
+        view.fill({'select_owner': owner,
+                   'select_group': group})
+        view.save_button.click()
+        view = self.create_view(MyServiceDetailView)
+        assert view.is_displayed
+        view.flash.assert_no_error()
+        view.flash.assert_success_message('Ownership saved for selected Service')
 
     def edit_tags(self, tag, value):
-        navigate_to(self, 'EditTags')
-        fill(edit_tags_form, {'select_tag': tag,
-                              'select_value': value},
-             action=form_buttons.save)
-        flash.assert_success_message('Tag edits were successfully saved')
+        view = navigate_to(self, 'EditTags')
+        view.fill({'select_tag': tag,
+                   'select_value': value})
+        view.save_button.click()
+        view = self.create_view(MyServiceDetailView)
+        assert view.is_displayed
+        view.flash.assert_no_error()
+        view.flash.assert_success_message('Tag edits were successfully saved')
 
     def check_vm_add(self, add_vm_name):
-        navigate_to(self, 'Details')
+        view = navigate_to(self, 'Details')
         quadicon = Quadicon(add_vm_name, "vm")
         sel.click(quadicon)
```

Comment:
Here should be a TODO that will let us know about these legacy things inside widgetastic.